4_Zadania_dodatkowe/4.ProgressBar/js/app.js

```diff
@@ -1,5 +1,5 @@
 $(document).ready(function () {
-   // console.log('test');
+    // console.log('test');
```

Comment:
why not remove? this line and other `console.log`s or redundant comments

Reply:
all redundant comments removed